apex-squid/src/test/java/org/fundacionjala/enforce/sonarqube/apex/parser/grammar/ApexGrammarDmlOperationTest.java

```diff
@@ -23,39 +23,35 @@
  */
 package org.fundacionjala.enforce.sonarqube.apex.parser.grammar;
 
+import org.fundacionjala.enforce.sonarqube.apex.parser.ApexRuleTest;
 import org.junit.Before;
 import org.junit.Test;
-
-import org.fundacionjala.enforce.sonarqube.apex.parser.ApexRuleTest;
-
 import static org.sonar.sslr.tests.Assertions.assertThat;
+import static org.fundacionjala.enforce.sonarqube.apex.api.grammar.ApexGrammarRuleKey.DML_OPERATION;
 
-import static org.fundacionjala.enforce.sonarqube.apex.api.grammar.ApexGrammarRuleKey.DML_STATEMENT;
-
-public class ApexGrammarDmlStamentTest extends ApexRuleTest {
-
-    @Before
+public class ApexGrammarDmlOperationTest extends ApexRuleTest{
+@Before
     public void init() {
-        setRootRule(DML_STATEMENT);
+        setRootRule(DML_OPERATION);
     }
 
     @Test
-    public void RulesDmlStament() {
+    public void RulesDmlOperation() {
```

Comment:
According to the Apex Grammar document the upsert and merge are declared not in the "DmlOperation" rule, but they have their own rules in the "DmlOperations" rule. 
<div id="centro2">
<p align="center"></p><h3>RULE</h3><p></p>
<p align="center"><b></b></p><h4><b>DML_OPERATIONS</b></h4><p></p>

<p align="left">
    DML_OPERATIONS::= <br>
    &nbsp;&nbsp;<a href="dmlOperation.html">DML_OPERATION</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="dmlUpsert.html">DML_UPSERT</a><br>
    &nbsp;&nbsp;|&nbsp;<a href="dmlMerge.html">DML_MERGE</a><br>
</p>
</div>